src/dcm/agent/cloudmetadata.py

```diff
@@ -35,11 +35,14 @@ class CLOUD_TYPES:
     CloudStack = "CloudStack"
     CloudStack3 = "CloudStack3"
     DigitalOcean = "DigitalOcean"
-    Eucalyptus = "Eucalyptus"
     Google = "Google"
     Joyent = "Joyent"
-    Konami = "Konami"
+    _Konami = "Konami" # won't show in interactive session
     OpenStack = "OpenStack"
+    VirtuStream = "VirtuStream"
+    ScaleMatrix = "ScaleMatrix"
+    VMware = "VMware"
+    WindowsAzurePack = "WindowsAzurePack"
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this field "WindowsAzurePack" to match the regular expression ^[_a-z][_a-z0-9]*$. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.int.enstratius.com/coding_rules#rule_key=python%3AS116)